benchmarks/src/test/java/org/apache/druid/benchmark/query/GroupByBenchmark.java

```diff
@@ -428,14 +424,12 @@ private void setupQueries()
   }
 
   @Setup(Level.Trial)
-  public void setup() throws IOException
+  public void setup()
   {
     log.info("SETUP CALLED AT " + +System.currentTimeMillis());
 
     ComplexMetrics.registerSerde("hyperUnique", new HyperUniquesSerde());
 
-    executorService = Execs.multiThreaded(numProcessingThreads, "GroupByThreadPool[%d]");
```

Comment:
from this point onward a bit hard to follow the reasoning for the changes? what part of the PR description does this relate to?

Reply:
These changes are due to the scoping of the benchmark. This setup method now only in charge of initializing everything common for benchmarking both the incremental-index and the queriable-index.
Anything specific to the incremental or queriable index was moved to its designated scope below.